src/controllers/livetv/livetvchannels.js

```diff
@@ -1,133 +1,134 @@
-                events.on(filterDialog, 'filterchange', function () {
-                    reloadItems(context);
-                });
-                filterDialog.show();
-            });
+        const query = getQuery();
+        context.querySelector('.paging').innerHTML = libraryBrowser.getQueryPagingHtml({
+            startIndex: query.StartIndex,
+            limit: query.Limit,
+            totalRecordCount: result.TotalRecordCount,
+            showLimit: false,
+            updatePageSizeSetting: false,
+            filterButton: false
+        });
+        const html = getChannelsHtml(result.Items);
+        const elem = context.querySelector('#items');
+        elem.innerHTML = html;
+        imageLoader.lazyChildren(elem);
+        let i;
+        let length;
+        let elems;
+
+        for (elems = context.querySelectorAll('.btnNextPage'), i = 0, length = elems.length; i < length; i++) {
+            elems[i].addEventListener('click', onNextPageClick);
         }
 
-        function reloadItems(context, save) {
-            loading.show();
-            isLoading = true;
-            var query = getQuery();
-            var apiClient = ApiClient;
-            query.UserId = apiClient.getCurrentUserId();
-            apiClient.getLiveTvChannels(query).then(function (result) {
-                renderChannels(context, result);
-                loading.hide();
-                isLoading = false;
-
-                require(['autoFocuser'], function (autoFocuser) {
-                    autoFocuser.autoFocus(view);
-                });
-            });
+        for (elems = context.querySelectorAll('.btnPreviousPage'), i = 0, length = elems.length; i < length; i++) {
+            elems[i].addEventListener('click', onPreviousPageClick);
         }
-
-        var pageData;
-        var self = this;
-        var isLoading = false;
-        tabContent.querySelector('.btnFilter').addEventListener('click', function () {
-            showFilterMenu(tabContent);
+    }
+
+    function showFilterMenu(context) {
+        import(['components/filterdialog/filterdialog']).then(({default: filterDialogFactory}) => {
+            const filterDialog = new filterDialogFactory({
+                query: getQuery(),
+                mode: 'livetvchannels',
+                serverId: ApiClient.serverId()
+            });
+            events.on(filterDialog, 'filterchange', function () {
+                reloadItems(context);
+            });
+            filterDialog.show();
         });
+    }
+
+    function reloadItems(context, save) {
+        loading.show();
+        isLoading = true;
+        const query = getQuery();
+        const apiClient = ApiClient;
+        query.UserId = apiClient.getCurrentUserId();
+        apiClient.getLiveTvChannels(query).then(function (result) {
+            renderChannels(context, result);
+            loading.hide();
+            isLoading = false;
+
+        import('autoFocuser').then(({default: autoFocuser}) => {
```

Comment:
```suggestion
            import('autoFocuser').then(({default: autoFocuser}) => {
```
Oops, i did the wrong indent